Make.inc

```diff
@@ -990,7 +998,7 @@ else ifneq ($(USEMSVC), 1)
 endif
 
 ifeq ($(OS), Linux)
-OSLIBS += -Wl,--no-as-needed -ldl -lrt -lpthread -Wl,--export-dynamic,--as-needed,--no-whole-archive $(LIBUNWIND)
+OSLIBS += -Wl,--no-as-needed -ldl -lrt -lpthread -Wl,--export-dynamic,--as-needed,--no-whole-archive
```

Comment:
Why remove `$(LIBUNWIND)` here?

Reply:
Well, it's not really needed for flisp, so it's a bit of a waste to build/link it there and it's not really an OS lib if we build it ourselves. You are right though that I forgot to add it back for libjulia.